perun/postprocess/regression_analysis/example/full_computation.perf

```diff
@@ -0,0 +1,322 @@
+{
+  "origin": "af10c9ae9eafec1e44cb48dbe498b513d679fbcd",
+  "header": {
+    "cmd": "complexity",
+    "workload": "",
+    "type": "mixed",
+    "units": {
+      "mixed(time delta)": "ms"
+    },
+    "params": ""
+  },
+  "postprocessors": [],
+  "collector_info": {
+    "name": "complexity",
+    "params": {
+      "sampling": [
+        [
```

Comment:
Can you elaborate, what is included in this list? I was wondering whether dictionary with named params would be a little bit more verbose. I mean, one of our goals is human readable format of profiles, so we should eventually get there :)

Reply:
Addressed in pull request #26 

changed to dictionary with more explicit keys.